src/pacer.js

```diff
@@ -151,6 +151,46 @@ let PACER = {
     return /iquery.pl/.test(url) && !/[?&]/.test(url)
   },
 
+  // Returns true if the page contains a Transaction Receipt table
+  hasTransactionReceipt: function(){
+    return !!$('th>font:contains("Transaction Receipt")').length;
+  },
```

Comment:
We do this a different way in toolbar_button.js, by checking the user's cookies. That determines something slightly different: Are receipts disabled globally? This determines if they're missing on a given page. 

Since the two things are so similar (but serve unique purposes), we should at least put them next to each other in our code base. Could you factor that one into a function called `areTransactionReceiptsDisabled`, and put it in this file?

Reply:
Yes! I will create this function and refactor the toolbar_button.js file.